consensus/pacemaker/pacemaker.go

```diff
@@ -253,12 +264,29 @@ func (p *paceMaker) startNextView(qc *typesCons.QuorumCertificate, forceNextView
 		}
 	}
 
-	p.RestartTimer()
-	p.consensusMod.broadcastToNodes(hotstuffMessage)
+	m.RestartTimer()
```

Comment:
Can you confirm there's a log line inside "RestartTimer"

Reply:
added